.build-harness

```diff
@@ -0,0 +1,19 @@
+export SHELL = /bin/bash
```

Comment:
Don't commit this file. Copy the `.gitignore` from the `terraform-example-module`